gpt_synthesizer/main.py

```diff
@@ -103,14 +162,20 @@ def main(verbose = False):
 
         code_generator(task_message, lang_message, comp, comp_list[comp],
                        generate_func_list_output, sum_output, llm, verbose)
+        
 
         all_comps_1, all_comps_2_spec, all_comps_2_func_list = make_func_list(comp, comp_list[comp],
                                                                               generate_func_list_output,
                                                                               all_comps_1,
                                                                               all_comps_2_spec,
                                                                               all_comps_2_func_list)
 
+    main_generator(task_message, lang_message, comp_list, sum_output, llm, verbose)
```

Comment:
Maybe activate this line only if the lang is python. We currently have not tested the other languages. Also, the code in generate_main.py only works for .py code files.